cwatm/hydrological_modules/water_demand/water_demand.py

```diff
@@ -661,6 +742,12 @@ def initial(self):
 
             self.var.act_nonpaddyConsumption = globals.inZero.copy()
             self.var.act_paddyConsumption = globals.inZero.copy()
+            
+            # for wastewater package
+            self.var.leakage_wwtC_daily = np.compress(self.var.compress_LR, globals.inZero.copy())
```

Comment:
self.var.compress_LR might not exist

Reply:
Fixed by including an 'if checkOption('includeWaterBodies'):' statement